plugin.audio.soundcloud/resources/lib/soundcloud/api_v2.py

```diff
@@ -0,0 +1,238 @@
+from future import standard_library
+standard_library.install_aliases()  # noqa: E402
+
+import hashlib
+import json
+import requests
+import urllib.parse
+import xbmc
+
+from resources.lib.models.playlist import Playlist
+from resources.lib.models.track import Track
+from resources.lib.models.selection import Selection
+from resources.lib.models.user import User
+from resources.lib.soundcloud.api_collection import ApiCollection
+from resources.lib.soundcloud.api_interface import ApiInterface
+
+
+class ApiV2(ApiInterface):
+    """This class uses the unofficial API used by the SoundCloud website."""
+
+    api_host = "https://api-v2.soundcloud.com"
+    api_client_id = "KT6UCHXC9iNnI8wn4UUfwMSlAPe4Z8zx"
```

Comment:
This API key changes regularly, I don't know how long it stays valid. But the key can be overridden via settings.

The key should be automatically extracted from the SoundCloud website. Otherwise the plugin has to be updated as soon as the key expires.